main.tf

```diff
@@ -80,6 +81,31 @@ data "aws_iam_policy_document" "default" {
       type        = "AWS"
       identifiers = ["*"]
     }
+  }]
+
+  statement = ["${flatten(data.aws_iam_policy_document.master.*.statement)}"]
+}
+
+data "aws_iam_policy_document" "master" {
+  count = "${signum(length(var.master_aws_account_ids))}"
+
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["${formatlist("arn:aws:iam::%v:root", var.master_aws_account_ids)}"]
+    }
+
+    actions = [
+      "s3:GetBucketVersioning",
+      "s3:PutBucketVersioning",
+      "s3:ReplicateObject",
+      "s3:ReplicateDelete",
+    ]
+
+    resources = [
+      "${aws_s3_bucket.default.arn}",
```

Comment:
In the original version, this was the source bucket arn. Now this is the destination bucket arn.

Reply:
https://github.com/cloudposse/terraform-aws-s3-website/pull/10/commits/96450a37a9b1909b855f0ad1ed120690493d7025#diff-7a370d8342e7203b805911c92454f0f4R105

Reply:
It was mistake